client/src/routes/login/tabs/SignIn.tsx

```diff
@@ -55,9 +55,7 @@ function SignIn() {
 			}
 		).then((res) => {
 			if(res.status == 200) {
-				console.log(res.data);
 				localStorage.setItem('token', res.data);
-				console.log(localStorage.getItem('token'));
 				
 				user.setUser({
 					email: email,
```

Comment:
We probably need more info from the user like the Id and name ect. the context is invisible to the end user so its safe.